### DISCUSSION TITLE
count number of walls.

### DISCUSSION THREAD

**Discussion by Chetan-Yeola** [mozman/ezdxf#781] [Q&A]
Can we count the number of walls present in the dxf file using ezdxf?

**Comment by f0rme** [comment]
How are you distinguishing the walls from the rest of the 'structure'?

**Comment by Chetan-Yeola** [comment]
Using Bounding box Library. 
![image](https://user-images.githubusercontent.com/72593205/204978084-71f37960-c174-4a7f-b010-54e8ff0c342c.png)

**Reply by f0rme** [reply]
It looks like LINE entities assigned the layer "Wall" are the 'walls' you're looking for.
Is that correct?

**Comment by Chetan-Yeola** [comment]
Yes.

**Comment by f0rme** [comment]
Perhaps I'm not understanding the question?

It seems like all you have to do is count the number of times your for loop runs.
